sway-lsp/test/fixtures/diagnostics/dead_code/Forc.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["Sophie"]
```

Comment:
Just noticed this.  We use `authors = ["Fuel Labs <contact@fuel.sh>"]` in the compiler tests, which is automatic if you set `FORC_INIT_MANIFEST_AUTHOR='Fuel Labs <contact@fuel.sh>'` in your env.

John used to be really strict with it but perhaps it isn't so important these days.

Reply:
Actually yeah it's probably better to be consistent and use `Fuel Labs`.

Reply:
Agree with the above.